### ISSUE TITLE
BUG - overlapping tiles on map - different size of tiles on map

### ISSUE THREAD

**Issue by DSuppo** [mapbox/mapbox-unity-sdk#984]
Hello, I've already posted this bug but I had no help on it.
But now I have one information more: this BUG seem to happen only with "world scale" option (that I need to use). I'm in Italy and it happen on each map I update through:

Mapbox.Unity.Map.AbstarctMap.UpdateMap(Vector2d latitudeLongitude, int zoomFactor);

You can see the problem quite well in the attached picture

Please if is not possible to solve this bug can you let me know how to restart the map from scratch with "custom size" option enabled, after that I've loaded the map with "world scale" option?

Many thanks!
Daniele

_Note: If this is a bug or support ticket, please provide the following information:_
* Unity version: 
  * Scripting Runtime Version: 
  * Scripting Backend: 
  * Api Compatibility Level: 
* Mapbox SDK version: 
* The platform you're building to:
* A description of what you're trying to do:
* Steps to recreate the bug if appropriate:
* Links to your logs are also helpful: 
![wrongmap2](https://user-images.githubusercontent.com/41523122/44676240-617a5680-aa3b-11e8-8753-51e16a30f2c8.jpg)

**Comment by atripathi-mb** [comment]
@DSuppo Can you share screenshots of your map settings + steps to reproduce the issue. 
At our end we are not able to reproduce the issue. Any details would be helpful to resolve the issue.

**Comment by jordy-isaac** [comment]
Closing this one for now. Please feel free to open with additional info.

**Comment by Pizoline** [comment]
This bug is still present in v2.1.1, and easy to reproduce:
-Create an empty scene
-Add a Mapbox Map
-Configure "Scaling Option" to "World Scale"
-Play
-During playmode, change location far away (using inspector or UpdateMap method)
Farther you jump, more is the tiles overlapping

**Comment by broseidon91** [comment]
For posterity, if you are seeing this issue on unity, I found a decent fix.

The fix is detailed in #1384 

Put simply, set your scaling strategy to "World Scale" in the map. 

Then, open `MapScalingAtWorldScaleStrategy.cs` and modify `SetupScaling()` to the following

```csharp
 var scaleFactor = Mathf.Pow(2, (map.AbsoluteZoom - map.InitialZoom));
map.SetWorldRelativeScale(scaleFactor);

```
